tools/deployment/package/docker-compose.base.yaml

```diff
@@ -35,23 +40,35 @@ x-volume-definitions:
     source: "${CLP_LOGS_DIR_HOST:-./var/log}/.clp-config.yml"
     target: "/etc/clp-config.yml"
     read_only: true
-  logs-input-readonly: &volume_root_logs_readonly
+  clp-logs: &volume_clp_logs
+    type: "bind"
+    source: "${CLP_LOGS_DIR_HOST:-./var/log}"
+    target: "/var/log"
+  logs-input-readonly: &volume_logs_input_readonly
     type: "bind"
     source: "${CLP_LOGS_INPUT_DIR_HOST:-/}"
     target: "${CLP_LOGS_INPUT_DIR_CONTAINER:-/mnt/logs}"
     read_only: true
 
+volumes:
+  # Dummy volume to use when a bind mount is not desired.
+  empty:
+    driver_opts:
+      device: "tmpfs"
+      type: "tmpfs"
+      size: 0
+
 services:
   database:
     <<: *service_defaults
-    image: "${CLP_DB_IMAGE:-mysql:8.0.23}"
+    image: "${CLP_DB_IMAGE:-mariadb:10-jammy}"
     hostname: "database"
-    user: "${CLP_SERVICE_CONTAINER_UID_GID:-1000:1000}"
+    user: "${CLP_THIRD_PARTY_SERVICE_UID_GID:-1000:1000}"
     environment:
-      MYSQL_DATABASE: "${CLP_DB_NAME}"
-      MYSQL_PASSWORD: "${CLP_DB_PASS}"
-      MYSQL_ROOT_PASSWORD: "${CLP_DB_PASS}"
-      MYSQL_USER: "${CLP_DB_USER}"
+      MYSQL_DATABASE: "${CLP_DB_NAME:-clp-db}"
+      MYSQL_PASSWORD: "${CLP_DB_PASS:?error}"
```

Comment:
Instead of "error", how about "Please set a value."? I don't think a more specific error would help since the error printed by Docker is already verbose (and we don't want to have to duplicate the error everywhere). Ditto everywhere else.